src/a.tex

```diff
@@ -232,13 +232,17 @@ \section{Load-Reserved/Store-Conditional Instructions}
 The LR/SC
 sequence can be given acquire semantics by setting the {\em aq} bit on
 the LR instruction.  The LR/SC sequence can be given release semantics
-by setting the {\em rl} bit on the SC instruction.  Setting the {\em
-  aq} bit on the LR instruction, and setting both the {\em aq} and the {\em
-  rl} bit on the SC instruction makes the LR/SC sequence sequentially
-consistent, meaning that it cannot be reordered with earlier or
-later memory operations from the same hart.
-
-If neither bit is set on both LR and SC, the LR/SC sequence can be
+by setting the {\em rl} bit on the SC instruction.  Setting the
+{\em aq} bit on the LR instruction, and setting the
+{\em rl} bit on the SC instruction potentially makes the LR/SC
```

Comment:
Why "potentially"? Is it possible to be more certain here, or at least explain what the uncertainty means?

Reply:
I'll clarify, at the expense of slightly higher verbosity.